doc/nrf/shortcuts.txt

```diff
@@ -164,6 +164,10 @@
 .. |application_sample_long_path_windows| replace:: On Windows, because of the `Windows path length limitations`_, the build can fail with errors related to permissions or missing files if some paths in the build are too long.
    To avoid this issue, shorten the build folder name, for example, from ``build_nrf5340dk_nrf5340_cpuapp_ns`` to ``build``, or shorten the path to the build folder in some other way.
 
+.. |common_sample_components_desc| replace:: Samples found in the :file:`samples/common` are not intended to be used independently, but in addition to other compatible samples for development purposes.
```

Comment:
```suggestion
.. |common_sample_components_desc| replace:: Code found in the :file:`samples/common` are not intended to be used independently, but in addition to other compatible samples for development purposes.
```

There are not standalone samples in this folder, as they can not be built.
Instead it is code used by other samples. I suggest a rewording to reflect this.